python/grass/jupyter/testsuite/display_test.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+
+############################################################################
+#
+# NAME:      display_test
+#
+# AUTHOR:    Caitlin Haedrich (caitlin dot haedrich gmail com)
+#
+# PURPOSE:   This is a test script for grass.jupyter's display module
+#
+# COPYRIGHT: (C) 2021 by Caitlin Haedrich and the GRASS Development Team
+#
+#            This program is free software under the GNU General Public
+#            License (>=v2). Read the file COPYING that comes with GRASS
+#            for details.
+#
+#############################################################################
+
+# Dependencies
+import os
+import unittest
+from pathlib import Path
+import grass.jupyter as gj
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+
+# Helper function
+def can_import_folium():
+    try:
+        import folium
+
+        return True
+    except ImportError:
+        return False
+
+
+# Tests
+class TestDisplay(TestCase):
+    # Setup variables
+    files = []
+
+    @classmethod
+    def setUpClass(cls):
+        """Ensures expected computational region"""
+        # to not override mapset's region (which might be used by other tests)
+        cls.use_temp_region()
+        # cls.runModule or self.runModule is used for general module calls
+        # we'll use the elevation raster as a test display
+        cls.runModule("g.region", raster="elevation")
+
+    @classmethod
+    def tearDownClass(cls):
+        """Remove temporary region"""
+        cls.del_temp_region()
+
+    def tearDown(self):
+        """
+        Remove the PNG file created after testing with "filename =" option.
+        This is executed after each test run.
+        """
+        for f in self.files:
+            Path(f).unlink(missing_ok=True)
+
+    @unittest.skipIf(can_import_folium() is False, "Cannot import folium")
+    def test_defaults(self):
+        """Test that GrassRenderer can create a map with default settings."""
+        # Create a map with default inputs
+        grass_renderer = gj.GrassRenderer()
+        # Adding vectors and rasters to the map
+        grass_renderer.run("d.rast", map="elevation")
+        grass_renderer.run("d.vect", map="roadsmajor")
+        # Assert image exists
```

Comment:
If the code below is not clear enough, I'm afraid that the word "assert" will not help much (not the most common word in English).

Reply:
Do you plan to update this? If you think it is good as is, just resolve this conversation.

Reply:
There were two `#Assert image exists` and I changed the wrong/other one. Oops! Now they both say something better. Thanks for catching it.